packages/node_modules/@webex/webex-core/test/unit/spec/interceptors/auth.js

```diff
@@ -124,15 +124,12 @@ describe('webex-core', () => {
             };
 
             webex.internal.services.waitForService = (pto) => Promise.resolve(
-              services[pto.name] ||
-              webex.internal.services.getServiceFromUrl(pto.url || '')
+              services[pto.name] || pto.url
```

Comment:
I know this isn't part of your PR, but this setup seems a bit strange. I would expect it to only be one resolution, then overridden in the test cases that are expecting the `pto.url` response.

Reply:
This mocks the function more accurately to how it works. This allows it to respond correctly in each potential use case throughout the tests. This stems from how integration tests don't work with our interceptor logic and must exist as unit tests.